src/main/java/hudson/plugins/sauce_ondemand/SauceEnvironmentUtil.java

```diff
@@ -209,4 +212,10 @@ public static String getBuildName(AbstractBuild<?, ?> build) {
         }
         return builder.toString();
     }
+
+    public static String generateTunnelIdentifier(final AbstractBuild build) {
```

Comment:
If I do this, then I need to modify all of the code to pass in the project name using the same line:
String rawName = build.getProject().getName();

Why not just leave this where it is instead of adding it to a bunch of other places?